contracts/SSVNetwork.sol

```diff
@@ -125,27 +126,34 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
     /* Operator External Functions */
     /*******************************/
 
+    function registerPrivateOperator(
```

Comment:
@lior-blox I like the idea of second function what do you think?

Reply:
@andrew-blox We decided to go with `registerOperator -> setOperatorWhitelist` to enable a private operator. thanks!